cairo1-run/README.md

```diff
@@ -16,7 +16,7 @@ Now that you have the dependencies necessary to run the tests, you can run:
 make test
 ```
 
-To execute a cairo 1 program
+To execute a cairo 1 program (you can also execute Sierra program)
```

Comment:
The program is still Cairo 1, right? Maybe rephrase that to "either as casm or Sierra"?

Reply:
Yes, but the command accepts now either Cairo source file or Sierra.